Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/BuildTopologyDictionary.h

```diff
@@ -68,15 +67,16 @@ class BuildTopologyDictionary
   friend std::ostream& operator<<(std::ostream& os, const BuildTopologyDictionary& BD);
   void printDictionary(std::string fname);
   void printDictionaryBinary(std::string fname);
+  void saveDictionaryRoot(const char* filename);
 
   int getTotClusters() const { return mTotClusters; }
   int getNotInGroups() const { return mNotInGroups; }
   int getNGroups() const { return mNumberOfGroups; }
 
  private:
   TopologyDictionary mDictionary; ///< Dictionary of topologies
-  std::map<unsigned long, std::pair<ClusterTopology, unsigned long>>
-    mTopologyMap; ///< Temporary map of type <hash,<topology,counts>>
+  std::unordered_map<unsigned long, std::pair<ClusterTopology, unsigned long>>
```

Comment:
In the previous version I've changed your unordered_map to map just because otherwise it was failing the MacOS test. With the reintroduction of the unordered_map, failure is now back. I am sure this is a root bug, will it work for the moment with map, until we sort out the unordered_map problem?